test/integration/specialist_document_test.rb

```diff
@@ -107,8 +107,8 @@ def test_meta(component)
   test "renders a nested contents list" do
     setup_and_visit_content_item('aaib-reports')
 
-    assert page.has_css?(".dash-list")
-    within ".dash-list" do
+    assert page.has_css?(".contents-list.contents-list-nested")
```

Comment:
Quite possibly, however this is a find/replace exercise to keep tests passing, rather than an assessment of every contents list test and its purpose.